autoload/neomake/makers/ft/python.vim

```diff
@@ -5,7 +5,10 @@ function! neomake#makers#ft#python#EnabledMakers()
         return s:python_makers
     endif
 
-    let makers = ['python', 'frosted']
+    let makers = ['python']
+    if executable('frosted')
+        call add(makers, 'frosted')
+    endif
```

Comment:
why do many of the other makers have this check here? It seems like all or none should have it.